networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -139,13 +139,61 @@ include::modules/nw-multus-tap-object.adoc[leveloffset=+2]
 
 * For more information about enabling an SELinux boolean on a node, see xref:../../nodes/nodes/nodes-nodes-managing.adoc#nodes-nodes-working-setting-booleans_nodes-nodes-managing[Setting SELinux booleans]
 
+// Set of includes pertains to OVN-Kubernetes additional network
 include::modules/configuring-ovnk-additional-networks.adoc[leveloffset=+2]
+
+[id="{context}_configuration-additional-network-types-supported-platforms"]
+==== Supported platforms for OVN-Kubernetes additional network
+
+You can use an OVN-Kubernetes additional network with the following supported platforms:
+
+- bare-metal
+- {ibmpowerProductName}
+- {ibmzProductName}
+- {linuxoneProductName}
+- VMware vSphere
+- {rh-openstack-first}
+
+// Workaround lack of xref in modules
+[id="{context}_configuration-additional-network-types-prerequisites"]
+==== Prerequisites for configuring OVN-Kubernetes additional network
+
+- Install NMstate Operator. For more information, see xref:../../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#"k8s-nmstate-about-the-k8s-nmstate-operator[About the Kubernetes NMState Operator]
+
+[id="{context}_configuration-additional-network-interface"]
+==== Configuration for an OVN-Kubernetes additional network mapping
+
+You must map the additional network to the OVN bridge to use it as an OVN-Kubernetes additional network. You can create an `NodeNetworkConfigurationPolicy` object, part of the `nmstate.io/v1` API group, to declaratively create the mapping.
+
+In the following example, the `localnet-network` additional network is attached to the `ovsbr1`. Through this attachment, the additional network is available to the OVN-Kubernetes network plugin as an additional network.
```

Comment:
I think these statements are too vague. I mean, it's generic enough to understand, but I think the reader needs to understand **why** this is needed.

If that makes sense for your (in context of our docs), you could maybe get some inspiration from [this openstack doc](https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.0/html/networking_guide/bridge-mappings) .

IMO these sentences (copied from the doc) are helpful - and don't go into details:
> Bridge mappings allow provider network traffic to reach the physical network.

> A bridge mapping associates a physical network name (an interface label) to a bridge created with OVS or OVN.

Finally, there's another reason why using the NNCP is a good call: you would need to do this on **all** worker nodes in the cluster. If you use a policy, you just apply it, and kubernetes-nmstate takes care of that for you.

Reply:
@maiqueb, I'm hampered by the fact that this is what I'd considered advanced networking, and my networking background is not deep enough to fully elucidate the particulars of what this feature actually does for a customer.

I'll look at the OpenStack link, thanks!